gr-blocks/lib/wavfile_sink_impl.cc

```diff
@@ -255,6 +265,20 @@ bool wavfile_sink_impl::stop()
         d_new_fp = nullptr;
     }
     close();
+    d_should_reopen = true;
+
+    return true;
+}
+
+bool wavfile_sink_impl::start()
+{
+    gr::thread::scoped_lock guard(d_mutex);
+    if (d_should_reopen) {
+        d_should_reopen = false;
+        if (!open_file_for_append()) {
+            d_logger->error("Failed to reopen output file after reconfiguration.");
+        }
```

Comment:
Can the sink continue to work in this situation?
If not, we should `return false` here!

Reply:
I agree, in this situation is more correct to return `false`.